lib/common/results.c

```diff
@@ -525,6 +526,7 @@ crm_exit_str(crm_exit_t exit_code)
         case CRM_EX_EXPIRED: return "Requested item has expired";
         case CRM_EX_NOT_YET_IN_EFFECT: return "Requested item is not yet in effect";
         case CRM_EX_INDETERMINATE: return "Could not determine status";
+        case CRM_EX_UNSATISFIED: return "Requested item does not satisfy constraints";
```

Comment:
To avoid confusion with user-configured constraints, I'd reword this, something like "Current conditions do not satisfy requirements" or "Not applicable under current conditions".

Reply:
Good point.  Fixed.